src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -171,13 +171,15 @@ void CoilCoolingDX::instantiateFromInputSpec(EnergyPlus::EnergyPlusData &state,
                                                                    DataLoopNode::ObjectIsNotParent);
 
     // Ultimately, this restriction should go away - condenser inlet node could be from anywhere
-    if (!OutAirNodeManager::CheckOutAirNodeNumber(state, this->condInletNodeIndex)) {
+    bool isOK = false;
+    OutAirNodeManager::CheckAndAddAirNodeNumber(state, this->condInletNodeIndex, isOK);
+    if (!isOK) {
         ShowWarningError(state,
                          std::string{routineName} + state.dataCoilCooingDX->coilCoolingDXObjectName + "=\"" + this->name + "\", may be invalid");
         ShowContinueError(state,
                           "Condenser Inlet Node Name=\"" + input_data.condenser_inlet_node_name +
                               "\", node does not appear in an OutdoorAir:NodeList or as an OutdoorAir:Node.");
-        ShowContinueError(state, "This node needs to be included in an air system or the coil model will not be valid, and the simulation continues");
+        ShowContinueError(state, "This node was added as an OutdoorAir:Node and the simulation continues");
```

Comment:
Final revisions: updated message.

Reply:
Looking at this again, Coil:Cooling:DX should not assume that the condenser inlet node is an outdoor air node. That defeats the flexibility of placing the condenser in a zone or and exhaust air stream or wherever the user wants. The general [node connection warnings](https://github.com/NREL/EnergyPlus/blob/4dfcba32e15375f1e1cf27b5d8b96b1c8845ed81/src/EnergyPlus/BranchNodeConnections.cc#L513-L561) will detect a dangling inlet node and throw a warning. This entire check should just go away. (lines 173:183 in this branch).

Reply:
Well doesn't that through a monkey wrench into things. If it is connected to another node then this will add as an OA node and write OA conditions there. And if not connected to another node and should be an OA node then no conditions will be written there. I'll remove this and see exactly what gets reported. I wonder how many of these warnings exist that should be reconsidered. The other comments will get cleaned up as well.